### PR TITLE
feat: upgrade amplitude sdk version to v8.19.2

### PR DESCRIPTION
**Pull request by shwetalodhe123**
# Description

- updated sdk version for Amplitude to latest version 8.19.2

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://github.com/CocoaPods/Specs.git'
 workspace 'RudderAmplitude.xcworkspace'
-use_frameworks!
+#use_frameworks!
```
Comment (1abhishekpandey) [inline Podfile:3]: Please un-comment this.

File: Podfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ end
 target 'RudderAmplitude' do
     project 'RudderAmplitude.xcodeproj'
     shared_pods
-    pod 'Amplitude', '8.8.0'
+    pod 'Amplitude', '8.19.2'
```
Comment (1abhishekpandey) [inline Podfile:14]: I think this is not needed. Please remove this.

File: Podfile (hunk 1 above)
Comment (shwetalodhe123) [reply Podfile]: done

File: Podfile (hunk 2 above)
Comment (shwetalodhe123) [reply Podfile]: done

**Review by 1abhishekpandey — APPROVED** [review]

**Pull request merged**